### ISSUE TITLE
User-Agent

### ISSUE THREAD

**Issue by null2264** [oscartbeaumont/ElectronPlayer#51]
Is there a way to change the user-agent permanently? I need it in order to open YouTube TV (i like to use it to just leanback and watch youtube without having to touch my mouse).

**Comment by oscartbeaumont** [comment]
Instead of exposing an option to modify the user-agent I think it would be best to just add a home screen button for Youtube TV and have it auto override. I will look into this for ElectronPlayer `v3` which will hopefully be out sometime soon. Which user-agent do you use, my research showed multiple and people reported some didn't work?

**Comment by null2264** [comment]
I'm using this user-agent "Mozilla/5.0 (DirectFB; Linux x86_64) Cobalt/40.13031-qa (unlike Gecko) Starboard/1"

**Comment by neogeo71** [comment]
I need a way to change it as youtue will not allow log in anymore unless i change user agent to:
Moillza/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/70.0.3538.102 Safari/537.36 Edge/18.18362

Is there a way to change it and compile myself?

**Comment by neogeo71** [comment]
i vote to have ability to update the user-agent permanently as well. I have to reset it using dev tools each time I watch DC Universe. Agent string I am using for DC Universe is: "Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/70.0.3538.102 Safari/537.36 Edge/18.18362"

**Comment by null2264** [comment]
Yeah, I also finding a way to update the user-agent by modifying the source... But i don't have the experience on this programming language.
And here's updated UserAgent string from LG : "Mozilla/5.0 (Unknown; Linux armv7l) AppleWebKit/537.1+ (KHTML, like Gecko) Safari/537.1+ LG Browser/6.00.00(+mouse+3D+SCREEN+TUNER; LGE; 42LA660S-ZA; 04.25.05; 0x00000001;); LG NetCast.TV-2013 /04.25.05 (LG, 42LA660S-ZA, wired)"
That may or may not helps the v3 development to put in Youtube TV

**Comment by dlenski** [comment]
> Instead of exposing an option to modify the user-agent I think it would be best to just add a home screen button for Youtube TV and have it auto override.

:100: agree with this.

The “YouTube™ for TV” browser extensions ([for Chrome](https://chrome.google.com/webstore/detail/youtube-for-tv/gmmbpchnelmlmndfnckechknbohhjpge?hl=en-US), [for FF](https://addons.mozilla.org/en-US/firefox/addon/youtube-for-tv/versions/) currently use this UA:

```
Mozilla/5.0 (SMART-TV; Linux; Tizen 5.0) AppleWebKit/538.1 (KHTML, like Gecko) Version/5.0 NativeTVAds Safari/538.1
```

**Comment by dlenski** [comment]
Looks like this was closed in 8914a52764a8184c68b2c8c3b4049a706be8153e already?
